### PR TITLE
self-managed docs rewrite

### PR DESCRIPTION
**Pull request by Hesperide**
Changes:
* Creates a new 'enterprise setup' section. I will add a section for Cloud (SSO) in a separate PR.
* Moves all self-managed docs into this section.
* Creates a more focused implementation guide, separated from existing OSS setup.
* Documents deploying self managed without SSO.
* Moves the instructions for creating a app for Airbyte in Okta (will be centralized after adding Cloud docs).

### CODE REVIEW

File: docs/enterprise-setup/self-managed/implementation-guide.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Implementation Guide
+
+[Airbyte Self Managed](./README.md) is in an early access stage for select priority users. Once you [are qualified for an Airbyte Self Managed license key](https://airbyte.com/company/talk-to-sales), you can deploy Airbyte with the following instructions.
```
Comment (colesnodgrass) [inline docs/enterprise-setup/self-managed/implementation-guide.md:6]: missing `-` in `Self-Managed`

File: docs/enterprise-setup/self-managed/implementation-guide.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+                <li>Install <a href="https://eksctl.io/introduction/">eksctl</a>.</li>
+                <li>Run <code>eksctl utils write-kubeconfig --cluster=$CLUSTER_NAME</code> to make the context available to kubectl.</li>
+                <li>Use <code>kubectl config get-contexts</code> to show the contexts available.</li>
+                <li>Run <code>kubectl config use-context $EKS_CONTEXT</code> to access the cluster with kubectl.</li>
+            </ol>
+        </TabItem>
+    </Tabs>
+</details>
+
+## Deploy Airbyte Self Managed
```
Comment (colesnodgrass) [inline docs/enterprise-setup/self-managed/implementation-guide.md:41]: Missing `-` in `Self-Managed`

File: docs/enterprise-setup/self-managed/implementation-guide.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+variable.
+
+### Customizing your Airbyte Self Managed Deployment
+
+In order to customize your deployment, you need to create `values.yaml` file in a local folder and populate it with default configuration override values. A `values.yaml` example can be located in [charts/airbyte](https://github.com/airbytehq/airbyte-platform/blob/main/charts/airbyte/values.yaml) folder of the Airbyte repository.
+
+After specifying your own configuration, run the following command:
+
+```text
+./tools/bin/install_airbyte_pro_on_helm.sh --values path/to/values.yaml %release_name% airbyte/airbyte
```
Comment (colesnodgrass) [inline docs/enterprise-setup/self-managed/implementation-guide.md:102]: I know this was in the previous docs as well, but using `%` here is incorrect.  We should have `$RELEASE_NAME` or simply leave it out.

File: docs/enterprise-setup/self-managed/sso.md [hunk 4]
```diff
@@ -1,28 +1,26 @@
...
-1. Configure Okta for Single Sign-On (SSO) with Airbyte Enterprise
-2. Deploy Airbyte Enterprise using Kubernetes (License Key Required)
+Airbyte Self Managed currently supports SSO via OIDC with [Okta](https://www.okta.com/) as an IdP. Support for Azure Active Directory and connecting via SAML are both coming soon. Please talk to us to learn more about upcoming [enterprise features](https://airbyte.com/company/talk-to-sales). 
 
-## Single Sign-On (SSO)
-
-Airbyte Enterprise supports Single Sign-On, allowing an organization to manage user access to their Airbyte Enterprise instance through the configuration of an Identity Provider (IdP). Airbyte Enterprise currently supports SSO via OIDC with [Okta](https://www.okta.com/) as an IdP.
+The following instructions walk you through:
+1. [Setting up the Okta OIDC App Integration to be used by your Airbyte instance](#setting-up-okta-for-sso)
+2. [Configuring Airbyte Self Managed to use SSO](#deploying-airbyte-enterprise-with-okta)
```
Comment (colesnodgrass) [inline docs/enterprise-setup/self-managed/sso.md:9]: Missing `-` in `Self-Managed`

File: docs/enterprise-setup/self-managed/sso.md [hunk 5]
```diff
@@ -42,12 +40,13 @@ Sign-out redirect URIs
...
 
-![Okta app integration name screenshot](./assets/docs/okta-login-redirect-uris.png)
+![Okta app integration name screenshot](../assets/okta-login-redirect-uris.png)
 
 _Example values_
 
 `<your-airbyte-domain>` should point to where your Airbyte instance will be available, including the http/https protocol.
 
+
 ## Deploying Airbyte Enterprise with Okta
```
Comment (colesnodgrass) [inline docs/enterprise-setup/self-managed/sso.md:50]: Seems we may be inconsistent with `Airbyte Self-Managed` vs `Airbyte Enterprise`.  Do we use both names or should be pick one?

**Review by colesnodgrass — APPROVED** [review]
With some recommendations to address a few inconsistencies.

File: docs/enterprise-setup/self-managed/implementation-guide.md (hunk 1 above)
Comment (Hesperide) [inline docs/enterprise-setup/self-managed/implementation-guide.md:6]:
```suggestion
[Airbyte Self-Managed](./README.md) is in an early access stage for select priority users. Once you [are qualified for an Airbyte Self Managed license key](https://airbyte.com/company/talk-to-sales), you can deploy Airbyte with the following instructions.
```

File: docs/enterprise-setup/self-managed/implementation-guide.md (hunk 2 above)
Comment (Hesperide) [inline docs/enterprise-setup/self-managed/implementation-guide.md:41]:
```suggestion
## Deploy Airbyte Self-Managed
```

File: docs/enterprise-setup/self-managed/implementation-guide.md (hunk 3 above)
Comment (Hesperide) [inline docs/enterprise-setup/self-managed/implementation-guide.md:102]:
```suggestion
./tools/bin/install_airbyte_pro_on_helm.sh --values path/to/values.yaml $RELEASE_NAME airbyte/airbyte
```

File: docs/enterprise-setup/self-managed/sso.md (hunk 4 above)
Comment (Hesperide) [inline docs/enterprise-setup/self-managed/sso.md:9]:
```suggestion
2. [Configuring Airbyte Self-Managed to use SSO](#deploying-airbyte-enterprise-with-okta)
```

File: docs/enterprise-setup/self-managed/sso.md (hunk 5 above)
Comment (Hesperide) [inline docs/enterprise-setup/self-managed/sso.md:50]:
```suggestion
## Deploying Airbyte Self-Managed with Okta
```

**Pull request merged**